geom/validation_test.go

```diff
@@ -8,10 +8,6 @@ import (
 	. "github.com/peterstace/simplefeatures/geom"
 )
 
-func xy(x, y float64) Coordinates {
```

Comment:
I think your recent changes may have made this redundant? if you feel it's still needed, I can add it back in.